<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Public Key</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_6">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Key Length:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLineEdit" name="len_edit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="public_key_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Private Key</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="private_key_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Plain Text</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="msg_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Encrypted Text</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="encrypted_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Decrypted Text</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="decrypted_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="gen_key_btn">
      <property name="text">
       <string>Generate Keys</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="crypt_btn">
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
